--- ipsorc-1.4/Makefile~	Sun Dec  2 18:28:20 2001
+++ ipsorc-1.4/Makefile	Sun Dec  2 18:28:20 2001
@@ -1,5 +1,6 @@
  ## Makefile for IPMagic ##
 
+FFLAGS = -g -O2
 CC = cc
 SPELL = magic
 SRC =  ipmagic.c spellbook.c cauldron.c spells.c tcpmagic.c
@@ -7,13 +8,13 @@
 #OBJS = $(SRC:.c=.o)
 
 con:
-	$(CC) -g -O2 -Wall -ansi $(CON) -o ipmagic
+	$(CC) $(CFLAGS) -Wall -ansi $(CON) -o ipmagic
 	
 gtk:
-	$(CC) -g -O2 -Wall -ansi $(SRC) -o $(SPELL) `gtk-config --cflags --libs`
+	$(CC) $(CFLAGS) -Wall -ansi $(SRC) -o $(SPELL) `gtk-config --cflags --libs`
    
 solaris:     
-	$(CC) -g -O2 -Wall -ansi $(CON) -o $(SPELL) -lsocket -lnsl
+	$(CC) $(CFLAGS) -Wall -ansi $(CON) -o $(SPELL) -lsocket
 
 gtk-install:
 	chmod 554 magic
