--- ipsorc-1.5/Makefile	Mon May  6 17:40:31 2002
+++ ipsorc-1.5/Makefile.org	Mon May  6 17:46:40 2002
@@ -11,16 +11,13 @@
 	@echo "Usage: make[gtk|con] then make [gtk-install|con-install]"
 
 con:
-	$(CC) -g -O2 -Wall -ansi -D_CON_ $(CON) -o ipmagic
-	
+	$(CC) $(CFLAGS) -Wall -ansi -D_CON_ $(CON) -o ipmagic	
 
 gtk:
-	$(CC) -g -O2 -Wall -ansi -D_GTK_ $(SRC) -o $(SPELL) `gtk-config --cflags --libs`
-
+	$(CC) $(CFLAGS) -Wall -ansi -D_GTK_ $(SRC) -o $(SPELL) `gtk-config --cflags --libs`
 
 solaris:     
-	$(CC) -g -O2 -Wall -ansi -D_CON_ $(CON) -o $(SPELL)  -lsocket -lnsl
-
+	 $(CC) $(CFLAGS) -Wall -ansi -D_CON_ $(CON) -o $(SPELL) -lsocket
 
 install: con-install
 
